worker/src/RTC/Transport.cpp

```diff
@@ -2371,7 +2371,7 @@ namespace RTC
 				// provide it with more available bitrate to choose its preferred layers.
 				for (uint8_t i{ 1u }; i <= priority; ++i)
 				{
-					uint32_t usedBitrate;
+					uint32_t usedBitrate = 0u;
```

Comment:
We use this everywhere AFAIR:

```suggestion
					uint32_t usedBitrate{ 0u };
```